### PR TITLE
add servicePrincipal to logs

### PR DESCRIPTION
**Pull request by ArvindHarinder1**

### CODE REVIEW

**Comment by Lauragra** [comment]
@ArvindHarinder1 , please set the ready to merge label when you want to publish. Thanks.

**Comment by Lauragra** [comment]
@ArvindHarinder1 , what is the merge status of this PR? Thanks.

**Comment by ArvindHarinder1** [comment]
@Lauragra  I'm still waiting for the API to be deployed. It got hung up.

**Comment by ArvindHarinder1** [comment]
@Lauragra the PR is okay to merge

File: api-reference/beta/api/provisioningobjectsummary-list.md [hunk 1]
```diff
@@ -187,6 +189,10 @@ Content-type: application/json
                     "newValue": "https://signin.aws.amazon.com/saml?metadata=aws|ISV9.1|primary|z"
                 },
             ],
+            "servicePrincipal": {
+                "id": "6cc35b93-185a-4485-a519-50c09549g3ad",
+                "displayName": "Amazon Web Services (AWS)"
```
Comment (Lauragra) [inline api-reference/beta/api/provisioningobjectsummary-list.md]: @ArvindHarinder1 , do we need to use a real company name in the example, or can we replace with something fictitious like Contoso? @davidmu1 , FYI.

**Pull request merged**